### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by kaggis**
New pull request for logging (due to too many merge conflicts of the last one)

### CODE REVIEW

**Comment by kaggis** [comment]
please review ./cc @pkoro

File: standalone/argorun.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env pythn
```
Comment (pkoro) [inline standalone/argorun.py]: minor typo in python

File: standalone/job_status_detail.py [hunk 2]
```diff
@@ -1,136 +1,160 @@
...
+    # Call pig
+    log.info("Submitting pig compute status detail job...")
+    run_cmd(cmd_pig, log)
+
+    # Cleaning hdfs sync data
+    if sync_clean == "true":
+        log.info("System configured to clean sync hdfs data after job")
+        run_cmd(cmd_clean_sync, log)
+
+    log.info("Excution of status job for tenant %s for date %s completed!",
```
Comment (pkoro) [inline standalone/job_status_detail.py]: minor typo in Excution

**Comment by pkoro** [comment]
+1

**Comment by pkoro** [comment]
+1

**Comment by skanct** [comment]
Implements part of ARGOeu/ARGO#88: 
- [x] Use syslog for central logging in the init scripts

**Pull request merged**